docs/systemuicontroller.md

```diff
@@ -0,0 +1,82 @@
+# System UI Controller for Jetpack Compose
+
+[![Maven Central](https://img.shields.io/maven-central/v/com.google.accompanist/accompanist-systemuicontroller)](https://search.maven.org/search?q=g:com.google.accompanist)
+
+System UI Controller provides easy-to-use utilities for updating the System UI bar colors within Jetpack Compose.
+
+## Usage
+To setup Insets in your composables, you need to call the provide an instance to the `LocalSystemUiController`
+composition local. This would typically be done near the top level of your composable hierarchy:
+
+``` kotlin
+setContent {
+    // Create a controller, and provide it to the LocalSystemUiController
+    val controller = androidSystemUiController(LocalView.current)
+    CompositionLocalProvider(LocalSystemUiController provides controller) {
+        MyHomeScreen()
+    }
+}
+```
+
+Then in your layouts, you can update the system bar colors as necessary like so:
+
+``` kotlin
+@Composable
+fun MyHomeScreen() {
+    // Get the current SystemUiController
+    val systemUiController = LocalSystemUiController.current
+    val isSystemInDarkTheme = isSystemInDarkTheme()
```

Comment:
What about querying the current theme? e.g.

```
val isDarkTheme = !MaterialTheme.colors.isLight
```

Reply:
Done